sc_rf_service/sc_rf_service.py

```diff
@@ -586,77 +591,77 @@ class CavityService(Service):
             if linac_name == "L1B":
                 cm_list += L1BHL
             for cm_name in cm_list:
-                
+
                 is_hl = cm_name in L1BHL
                 heater_prefix = f"CPIC:CM{cm_name}:0000:EHCV:"
                 self.add_pvs(HeaterPVGroup(prefix=heater_prefix))
-                
+
                 self[f"CRYO:CM{cm_name}:0:CAS_ACCESS"] = ChannelEnum(enum_strings=("Close", "Open"),
                                                                      value=1)
                 self[f"ACCL:{linac_name}:{cm_name}00:ADES_MAX"] = ChannelFloat(value=168.0)
-                
+
                 for cav_num in range(1, 9):
                     cm_prefix = f"ACCL:{linac_name}:{cm_name}"
                     cav_prefix = cm_prefix + f"{cav_num}0:"
-                    
+
                     jt_prefix = f"CLIC:CM{cm_name}:3001:PVJT:"
                     liquid_level_prefix = f"CLL:CM{cm_name}:"
-                    
+
                     HOM_prefix = f"CTE:CM{cm_name}:1{cav_num}"
                     cryo_prefix = f"CLL:CM{cm_name}:2601:US:"
-                    
+
                     cavityGroup = CavityPVGroup(prefix=cav_prefix, isHL=is_hl)
                     self.add_pvs(cavityGroup)
                     self.add_pvs(SSAPVGroup(prefix=cav_prefix + "SSA:",
                                             cavityGroup=cavityGroup))
-                    
+
                     self.add_pvs(PiezoPVGroup(prefix=cav_prefix + "PZT:"))
                     self.add_pvs(StepperPVGroup(prefix=cav_prefix + "STEP:",
                                                 cavity_group=cavityGroup))
                     self.add_pvs(CavFaultPVGroup(prefix=cav_prefix))
-                    
+
                     self.add_pvs(JTPVGroup(prefix=jt_prefix))
                     self.add_pvs(LiquidLevelPVGroup(prefix=liquid_level_prefix))
-                    
+
                     # Rack PVs are stupidly inconsistent
                     if cav_num in rackA:
                         hwi_prefix = cm_prefix + "00:RACKA:"
                     else:
                         hwi_prefix = cm_prefix + "00:RACKB:"
-                    
+
                     self.add_pvs(HWIPVGroup(prefix=hwi_prefix))
                     self.add_pvs(BeamlineVacuumPVGroup(prefix=cm_prefix + "00:"))
                     self.add_pvs(CouplerVacuumPVGroup(prefix=cm_prefix + "10:"))
                     self.add_pvs(CryomodulePVGroup(prefix=cm_prefix + "00:"))
                     self.add_pvs(HOMPVGroup(prefix=HOM_prefix))
                     self.add_pvs(CryoPVGroup(prefix=cryo_prefix))
-        
+
         self["ACCL:L2B:1400:RACKA:HWINITSUM"].write(3, severity=AlarmSeverity.INVALID_ALARM)
 
 
 def main():
     service = CavityService()
     get_event_loop()
     _, run_options = ioc_arg_parser(
-            default_prefix='',
-            desc="Simulated CM Cavity Service")
+        default_prefix='',
```

Comment:
This happened because of the auto formatting, that's why it looks like you have a million changes